packages/kit/src/views/BulkCopyAddresses/pages/BulkCopyAddresses.tsx

```
@@ -0,0 +1,680 @@
+      name,
+      value,
+      intRequired,
+    }: {
+      name: string;
+      value: string | undefined;
+      intRequired?: boolean;
+    }) => {
+      const filedName = name as keyof typeof formRangeWatchFields;
+      const valueBN = new BigNumber(value ?? 0);
+      if (valueBN.isNaN()) {
+        const formattedValue = parseFloat(value ?? '');
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)